src/haddock/modules/scoring/emscoring/cns/scoring.cns

```diff
@@ -55,6 +55,12 @@ evaluate ($Data.flags.vean = false)
 
 evaluate ($data.ncomponents=$ncomponents)
 
+evaluate ($nmol=1)
```

Comment:
I don't see why this would be needed? Were there errors related to that? We read by default all param files - see `read_param.cns`

Reply:
At some point CNS is also looking for this:
```
     do (store6 = 1) (tag and segid $Toppar.prot_segid_$nchain1 
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```